lib/update.js

```diff
@@ -1,5 +1,11 @@
 import isPlainObject from 'lodash/isPlainObject'
+import _omitBy from 'lodash/omitBy'
+
 import wrap from './wrap'
+import constant from './constant'
+
+const innerOmitted = constant('omitted')
```

Comment:
I think this can just be `{}` since that will work for reference equality. Making it a function that returns the string `omitted` doesn't seem necessary unless I'm missing something.

Reply:
Perhaps something like `{ __omitted: true }`. Still a reference, just more meaningful if seen in debug

Reply:
Also, there are some dangers to relying on reference. If a person has two versions of updeep in their tree, it's possible that this won't work properly. Maybe it should be modeled after however lodash does their placeholder? I don't remember how off the top of my head.

Reply:
[lodash] Hmmm... I'll have a look.

Reply:
Although I think the ref won't work only if somebody does:

```
import u1 from 'updeep';
import u2 from './path/to/other/updeep';

u1({ foo: u2.omitted })(x);
```

which would be rather twisted.

Reply:
agree that it's unlikely, if someone has a library that depends on updeep to build custom operators you could get into this situation. That library may export an operator that uses `omitted` from its version of `updeep` and if you use it with yours...

Not likely, I agree, but if lodash has an easy way to deal w/ it, it's probably good to follow that pattern.

Reply:
And unless I'm mistaken, lodash does it by attaching a `placeholder` attribute to the main exported `_` variable: https://github.com/lodash/lodash/blob/4ea8c2ec249be046a0f4ae32539d652194caf74f/.internal/getHolder.js

We should be able to do the same. Give me 5 minutes. :-)

Reply:
Darn. `update` is wrapped via `wrap` before it's exported, so binding would be a little more involved than I thought. And I'm not sure it'd bring any more safety than using the `omitted` and `update` from the same `u` source.

Reply:
okay, let's just try a reference for now.